src/workers/cdc.ts

```diff
@@ -292,20 +295,50 @@ const onPostChange = async (
     if (data.payload.after.visible) {
       await notifyPostVisible(logger, data.payload.after);
     }
+    if (data.payload.after.type === PostType.Freeform) {
+      const freeform = data as ChangeMessage<FreeformPost>;
+      if (
+        freeform.payload.after.content.length >=
+        FREEFORM_POST_MINIMUM_CONTENT_LENGTH
+      ) {
+        await notifyContentRequested(logger, {
+          id: freeform.payload.after.id,
+          content: freeform.payload.after.content,
+          post_type: freeform.payload.after.type,
+        });
+      }
+    }
   } else if (data.payload.op === 'u') {
     if (data.payload.after.visible) {
       if (!data.payload.before.visible) {
         await notifyPostVisible(logger, data.payload.after);
       } else {
         // Trigger message only if the post is already visible and the conte was edited
-        const freeform = data as unknown as ChangeMessage<FreeformPost>;
+        const freeform = data as ChangeMessage<FreeformPost>;
         if (
           isChanged(freeform.payload.before, freeform.payload.after, 'content')
         ) {
           await notifyPostContentEdited(logger, data.payload.after);
         }
       }
     }
+
+    if (data.payload.after.type === PostType.Freeform) {
+      const freeform = data as unknown as ChangeMessage<FreeformPost>;
```

Comment:
you probably also don't need the `as unknown` here.
Really can't recall why it's introduced 😅